src/views/docs/en/reference/project-manifest/tables-streams.md

```diff
@@ -0,0 +1,132 @@
+---
+title: '@tables-streams'
+category: app.arc
+description: Define DynamoDB tables with streaming changes
+---
+
+Define Lambda functions for streaming changes to DynamoDB tables. Respond to `insert`, `update`, and `destroy` events with a handler function.
+
+> ⚠️  Unfortunately, "dynalite" (used under the hood in Architect [Sandbox](../cli/sandbox)) doesn't support streams, so emulation isn't yet supported for local development.
+
+## Recommended Resources
```

Comment:
Personally I prefer to put these kinds of things at the bottom unless they are required reading.

Reply:
Agreed. it kind of interrupts the doc too much